staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -461,14 +462,14 @@ func (a *Admission) EvaluatePod(ctx context.Context, nsPolicy api.Policy, nsPoli
 
 	auditAnnotations := map[string]string{}
 	if nsPolicyErr != nil {
-		klog.V(2).InfoS("failed to parse PodSecurity namespace labels", "err", nsPolicyErr)
+		logger.V(2).Info("failed to parse PodSecurity namespace labels", "err", nsPolicyErr)
 		auditAnnotations["error"] = fmt.Sprintf("Failed to parse policy: %v", nsPolicyErr)
 		a.Metrics.RecordError(false, attrs)
 	}
 
-	klogV := klog.V(5)
+	klogV := logger.V(5)
 	if klogV.Enabled() {
```

Comment:
nit: inline this in the if to keep it scoped here
```suggestion
	if klogV := logger.V(5); klogV.Enabled() {
```

Reply:
That is indeed the recommended pattern.